client.lua

```diff
@@ -99,17 +49,21 @@ RegisterKeyMapping('scoreboard', 'Open Scoreboard', 'keyboard', Config.OpenKey)
 -- Threads
 
 CreateThread(function()
+    local loop = 1000
     while true do
         if scoreboardOpen then
-            for _, player in pairs(GetPlayersFromCoords(GetEntityCoords(PlayerPedId()), 10.0)) do
+            for _, player in pairs(QBCore.Functions.GetPlayersFromCoords(GetEntityCoords(PlayerPedId()), 10.0)) do
                 local PlayerId = GetPlayerServerId(player)
                 local PlayerPed = GetPlayerPed(player)
                 local PlayerCoords = GetEntityCoords(PlayerPed)
                 if Config.ShowIDforALL or PlayerOptin[PlayerId].permission then
-                    DrawText3D(PlayerCoords.x, PlayerCoords.y, PlayerCoords.z + 1.0, '['..PlayerId..']')
+                    QBCore.Functions.DrawText3D(PlayerCoords.x, PlayerCoords.y, PlayerCoords.z + 1.0, '['..PlayerId..']')
                 end
             end
+            loop = 0
```

Comment:
Place this inside the if statement instead of outside it so it doesn't wait 0 for nothing